framework/include/base/SystemBase.h

```diff
@@ -568,24 +568,39 @@ class SystemTempl : public SystemBase
   virtual DofMap & dofMap() { return _sys.get_dof_map(); }
   virtual System & system() { return _sys; }
 
-  virtual void addVariableToCopy(const std::string & dest_name, const std::string & source_name, unsigned int timestep)
+  virtual void addVariableToCopy(const std::string & dest_name, const std::string & source_name, const std::string & timestep)
   {
     _var_to_copy.push_back(VarCopyInfo(dest_name, source_name, timestep));
   }
 
   void copyVars(ExodusII_IO & io)
   {
+    int n_steps = io.get_num_time_steps();
+
     bool did_copy = false;
     for (std::vector<VarCopyInfo>::iterator it = _var_to_copy.begin();
         it != _var_to_copy.end();
         ++it)
     {
-      did_copy = true;
       VarCopyInfo & vci = *it;
+      int timestep = -1;
+
+      if (vci._timestep == "END")
```

Comment:
Good catch. I changed my mind what to call it and the input file didn't use
the default.

On Tue, Sep 29, 2015 at 4:27 PM Daniel Schwen <notifications@github.com>
wrote:

> In framework/include/base/SystemBase.h
> <https://github.com/idaholab/moose/pull/5754#discussion_r40738830>:
>
> >        VarCopyInfo & vci = *it;
> > +      int timestep = -1;
> > +
> > +      if (vci._timestep == "END")
>
> Default below is "LAST"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5754/files#r40738830>.
>